README.md

```diff
@@ -1,4 +1,16 @@
 [![Python package](https://github.com/mobiusml/aana_sdk/actions/workflows/python-package.yml/badge.svg)](https://github.com/mobiusml/aana_sdk/actions/workflows/python-package.yml)
+![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)
+![Build Status](https://img.shields.io/badge/build-passing-brightgreen.svg)
```

Comment:
This doesn't make sense. It shows that it's always passing. Maybe let's rename "Python package" badge to "Build Status" and remove that one.

Reply:
Done

Reply:
It still shows up as a "Python package" but I guess it's alright.